src/climate_learn/loaders.py

```diff
@@ -0,0 +1,356 @@
+# Standard library
+from typing import Any, Callable, Dict, Iterable, Optional, Tuple, Union
+from functools import partial
+import warnings
+
+# Local application
+from .data import DataModule, IterDataModule
+from .models import MODEL_REGISTRY, LitModule
+from .models.hub import (
+    Climatology,
+    Persistence,
+    LinearRegression,
+    ResNet,
+    Interpolation
+)
+from .models.lr_scheduler import LinearWarmupCosineAnnealingLR
+from .transforms import TRANSFORMS_REGISTRY
+from .metrics import METRICS_REGISTRY, MetricsMetaInfo
+
+# Third party
+import torch
+import torch.nn as nn
+import torch.optim.lr_scheduler as LRScheduler
+
+
+def load_model_module(
+    task: str,
+    data_module: Union[DataModule, IterDataModule],
+    preset: Optional[str] = None,
+    model: Optional[Union[str, nn.Module]] = None,
+    model_kwargs: Optional[Dict[str, Any]] = None,
+    optim: Optional[Union[str, torch.optim.Optimizer]] = None,
+    optim_kwargs: Optional[Dict[str, Any]] = None,
+    sched: Optional[Union[str, LRScheduler._LRScheduler]] = None,
```

Comment:
`LRScheduler` is an alias for `torch.topim.lr_scheduler`, which is a module. The name of the actual learning rate scheduler base class is `_LRScheduler`. I think what you're suggesting is changing
```python
import torch.optim.lr_scheduler as LRScheduler
```
to
```python
from troch.optim.lr_scheduler import _LRScheduler as LRScheduler
```
This can be done. I'll make the changes shortly.

Reply:
> `LRScheduler` is an alias for `torch.topim.lr_scheduler`, which is a module. The name of the actual learning rate scheduler base class is `_LRScheduler`.

I didn't knew that. 




> 
> ```python
> from troch.optim.lr_scheduler import _LRScheduler as LRScheduler
> ```

That would be perfect.